nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -103,7 +103,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer, no_lm_init=True):
         self.tokenizer = None
 
         with open_dict(cfg):
-            if cfg.get('precision', None) is None and trainer is not None:
```

Comment:
The latter condition is guaranteed by [the check in L93](https://github.com/NVIDIA/NeMo/blob/52d50e9e09a3e636d60535fd9882f3b3f32f92ad/nemo/collections/nlp/models/language_modeling/megatron_base_model.py#L93)